oci/oci_unix.go

```diff
@@ -0,0 +1,25 @@
+package oci
```

Comment:
Don't we need explicit build constraints in this file (`// +build …`).  The `oci_unix.go` doesn't match [the implicit-constraint `*_GOOS` pattern][1], because `unix` is not a [recognized `GOOS`][2].  For example, with this dummy project:

```console
$ head *.go
==> main.go <==
package main

import "fmt"

func main() {
        fmt.Println(nonWindows)
}

==> main_unix.go <==
package main

import "golang.org/x/sys/unix"

var nonWindows = unix.MS_LAZYTIME
$ GOOS=windows go build ./
# _/tmp/x
./main_unix.go:5:18: undefined: unix.MS_LAZYTIME
```

you can see that `GOOS=windows go build ./` is still attempting to compile `main_unix.go` despite the filename.
[1]: https://golang.org/pkg/go/build/#hdr-Build_Constraints
[2]: https://golang.org/doc/install/source#environment

Reply:
@vbatts Can you just add a !windows tag?

Reply:
i think so